Utils/utils.py

```
@@ -597,12 +616,31 @@ def is_url(url):
 
 # --- LLM Generation Utilities ---
 
-def gemini_generate(request: str | dict, model_name: str = "gemini-2.0-flash", gen_config: dict = None) -> str | None:
+def gemini_generate(
+    request: str | dict,
+    model_name: str = "gemini-flash-lite-latest",
+    gen_config: dict = None,
```

Ruff (0.15.18)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)